crates/gpui/src/platform/linux/platform.rs

```diff
@@ -340,7 +340,9 @@ impl Platform for LinuxPlatform {
     }
 
     //todo!(linux)
```

Comment:
Let's remove this todo, since it's done!

Reply:
Fixed.